### DISCUSSION TITLE
Blank bar on the Spectrogram

### DISCUSSION THREAD

**Discussion by KyuubiYoru** [coqui-ai/TTS#1921] [General Q&A]
What is the cause of the blank bar on the right side?
I not sure how i can fix that.

![image](https://user-images.githubusercontent.com/33806787/187093841-7eb17f0a-b529-48f1-8c9e-a58950ea0d1f.png)

![image](https://user-images.githubusercontent.com/33806787/187094013-90e9f5f2-3542-4165-995b-02b2c399cfdb.png)

**Comment by KyuubiYoru** [comment]
I changed the audio settings for signal normalization and now it looks like this.

![image](https://user-images.githubusercontent.com/33806787/187166519-e02116f4-481b-49c2-9809-832f37ff0622.png)

**Reply by p0p4k** [reply]
Blank bar is the padding. 
When training, the model finds out the longest audio length and pads all other audio spectrograms to make all of them the same length. There is nothing wrong in here.
